.core_files.yaml

```diff
@@ -6,7 +6,7 @@ core: &core
   - homeassistant/helpers/*
   - homeassistant/package_constraints.txt
   - homeassistant/util/*
-  - pyproject.yaml
+  - pyproject.toml
```

Comment:
One might consider this should be in a separate PR ;)
But I think it's OK here.

Reply:
> One might consider this should be in a separate PR ;)

Would have done so normally. It fits in here though.